src/models/inbound/consentRequests.model.ts

```diff
@@ -99,13 +100,11 @@ export class InboundConsentRequestsRequestModel {
       this.logger.push({ err }).error('Error in patchConsentRequest @ Inbound')
       const mojaloopError = this.reformatError(err)
```

Comment:
where `reformatError` function is defined in `InboundConsentRequestsRequestModel` class?
@sridharvoruganti ^^^

Reply:
there should be a unit test for consentRequests.model.ts: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/test/unit/models/inbound/consentRequests.model.test.ts
which covers unhappy scenarios and proper use of putConsentRequestsError